constants/Colors.js

```diff
@@ -5,19 +5,22 @@
  */
 const backgroundColor = '#101010';
 const textColor = '#fff';
-const tintColor = '#00a4dc';
+const jellyBlue = '#00a4dc';
+const jellyPurple = '#AA5CC3';
```

Comment:
I like to avoid product names in the codebase to keep everything generic, just a preference.

Reply:
Good point! They have been updated!